index.js

```diff
@@ -498,23 +504,23 @@ input DuplicateSectionInput {
 }
 
 input CreateIntroductionInput {
```

Comment:
Not sure but I have a feeling these Input types will need to be deprecated as well so they can be deleted later?

Reply:
Removed on deprecated fields branch